plugin/src/main/kotlin/tanvd/grazi/utils/TextUtils.kt

```diff
@@ -15,6 +17,11 @@ object Text {
 
     fun isHiddenFile(str: String) = str.startsWith(".")
     fun isHtmlUnicodeSymbol(str: String) = str.startsWith("&")
+
+    private val levenshtein = LevenshteinDistance()
```

Comment:
I would create sub object here `object Levenstein` and call it `Text.Levenstein.distance(...)`

Reply:
Done